### ISSUE TITLE
[Bad Example] Function names should say what they do

### ISSUE THREAD

**Issue by peter-gribanov** [jupeter/clean-code-php#23]
[Function names should say what they do](https://github.com/jupeter/clean-code-php#function-names-should-say-what-they-do)

**Bad:**
```php
function addToDate(\DateTime $date, $month) {
    $date->modify(sprintf('+%d months', $month));
}

$date = new \DateTime();

// It's hard to to tell from the function name what is added
addToDate($date, 1);
```

**Good**:
```php
function addMonthToDate($month, \DateTime $date) {
    $date->modify(sprintf('+%d months', $month));
}

$date = new \DateTime();
addMonthToDate(1, $date);
```

What is the purpose of `addToDate()`/`addMonthToDate()` functions?
What are they needed for?
These functions are meaningless.

**Comment by hailwood** [comment]
perhaps something more like

```php
class Email 
{
    //...

    public function handle() {
        mail($this->to, $this->subject, $this->body);
    }

}

$message = new Email(...);
// What is this? A handle for the message? Are we writing to a file now?
$message->handle();
```

compared to

```php
class Email 
{
    //...

    public function send() {
        mail($this->to, $this->subject, $this->body);
    }

}

$message = new Email(...);
// Clear and obvious
$message->send();
```

**Comment by peter-gribanov** [comment]
@hailwood your example i like more

**Comment by piotrplenik** [comment]
@hailwood Could you create PR for that?

**Comment by TomasVotruba** [comment]
Closed by #44
